mission/eventhandlers/mission/eh_EntityRespawned.sqf

```diff
@@ -73,7 +73,7 @@ if (isPlayer _entity) then
 		}, [_entity], 1] call CBA_fnc_waitAndExecute;
 	};
 
-	[_entity] call vn_mf_fnc_curator_init;
+	[_entity] call vn_mf_fnc_curator_init_client;
```

Comment:
this event handler is called server side? no?

Reply:
ah, doesn't matter because `vn_mf_fnc_curator_init_client` doesn't need `player` object

`_entity` is the server tracking the player.

although, could this be the source of the curator lost on respawn bug? it's a transient bug tho...